package-lock.json

```diff
@@ -3525,7 +3525,7 @@
       "resolved": "https://registry.npmjs.org/@types/redux-mock-store/-/redux-mock-store-1.0.1.tgz",
       "integrity": "sha512-1egEnh2/+sRRKImnCo5EMVm0Uxu4fBHeLHk/inhSp/VpE93It8lk3gYeNfehUgXd6OzqP5LLA9kzO9x7o3WfwA==",
       "requires": {
-        "redux": "^4.0.4"
```

Comment:
Can you revert these `package-lock.json` changes? They aren't needed for your PR.

Reply:
Whoops, sorry. Didn't realize that got committed.

Reverted the change
packge-lock.json is not longer part of the PR.